plugins/auth-backend/src/service/router.ts

```diff
@@ -158,3 +161,22 @@ export async function createRouter({
 
   return router;
 }
+
+export function createOriginFilter(
+  config: Config,
+): (origin: string) => boolean {
+  const allowedOrigins = config.getOptionalStringArray(
+    'auth.experimentalExtraAllowedOrigins',
+  );
+  if (!allowedOrigins || allowedOrigins.length === 0) {
+    return () => false;
```

Comment:
Should this return `true`? This is a breaking change otherwise, I got this rejection locally after pulling this in:

![origin-not-allowed](https://user-images.githubusercontent.com/556258/130125194-43409df3-b5c1-470a-9611-9fefb3860ca5.png)

It seems that the `auth.experimentalExtraAllowedOrigins` is required now?